src/transformState.js

```diff
@@ -5,7 +5,27 @@
  * @param {Object[]} actions
  */
 function transformState(state, actions) {
-  // write code here
+  for (const action of actions) {           //sort out an array with actions 
```

Comment:
if code is written properly, there is no need to add comments, avoid doing so
generally, if some code is required to have descriptions, docs are created